.github/workflows/combine-prs.yml

```diff
@@ -66,6 +71,24 @@ jobs:
                     }
                   }
                 }
+                if(${{ github.event.inputs.mustBeUnblocked }}) {
+                  console.log('Checking blocked status: ' + pull['number']);
+                  const labels = await github.paginate('GET /repos/{owner}/{repo}/issues/{issue_number}/labels', {
+                    owner: context.repo.owner,
+                    repo: context.repo.repo,
+                    issue_number: pull['number']
+                  });
+                  if(labels.length > 0) {
+                    for (const label of labels) {
+                      const label_name = label['name'];
+                      console.log('Validating label: ' + label_name);
+                      if(label_name == 'blocked') {
+                        console.log('Discarding ' + branch + ' with blocked label on PR');
+                        statusOK = false;
+                      }
+                    }
+                  }
+                }
```

Comment:
Might be good to shortcut this if `statusOK` is already false, so we don't consume too much API quota on a check that doesn't matter.